conf/spark-env.sh

```diff
@@ -69,7 +70,22 @@ if [[ -n "${KRB5CONF}" ]]; then
     echo "spark-env: Copying krb config from $KRB5CONF to /etc/" >&2
     echo "${KRB5CONF}" | ${BASE64_D} > /etc/krb5.conf
 else
-    echo "spark-env: No SPARK_MESOS_KRB5_CONF_BASE64 decoded" >&2
+    echo "spark-env: No SPARK_MESOS_KRB5_CONF_BASE64 to decode" >&2
+fi
+
+if [ -n "${STATSD_UDP_HOST}" ] && [ -n "${STATSD_UDP_PORT}" ]; then
```

Comment:
This change here adds metrics too. Is that ready for this release?

Reply:
Yes, product signed off and Nick created an upstream PR to update metrics too: https://github.com/apache/spark/pull/21516